test/integration/setup.js

```diff
@@ -275,8 +299,10 @@ async function setup() {
   };
 
   await cover.changeDependentContractAddress();
+  await stakingProducts.changeDependentContractAddress();
+  await coverProducts.changeDependentContractAddress();
```

Comment:
Is this actually needed?

Reply:
Yes if you remove those 2 lines - the tests start failing.

The storeddependencies are not there (0 address) so transactions fail.

Reply:
It's because contracts are added using the function initialize from our disposable:

https://github.com/NexusMutual/smart-contracts/blob/b4a1d0a51270eb817cfc67baa88810497cd9d4a5/contracts/mocks/Disposables/DisposableNXMaster.sol#L9


Not added through the standard function.